master/getting-started/kubernetes/upgrade/upgrade.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Upgrading Calico components
+no_canonical: true
+---
+  
+
+1. Upgrade each component to the latest v3.0.x release.
```

Comment:
Should you be using some clever {{ version }} substitution here - so that in future this could automatically say (e.g.) `Upgrade each component to the latest v3.2.x release` ?

Reply:
(If so, probably want to review everywhere in this PR where you've written `3.0`.)

Reply:
Yes, I will look into this further. 👍